### DISCUSSION TITLE
How about Arabic !?

### DISCUSSION THREAD

**Discussion by amans199** [vuejs-translations/guidelines#1] [Ideas]
Is there anyone working on translating the Vue docs to Arabic !?

If there is no one, I will be more than thrilled to start translating it.

**Comment by boussadjra** [comment]
@amans199 that's great, I'll support you and I could take some tasks to create this translation.

**Reply by amans199** [reply]
Awesome, 
We can wait until @yyx990803 creates the guidelines for the community translations, then we can start.

**Comment by theadell** [comment]
I would love to help as well! Are there any updates as when this will get started?

**Reply by amans199** [reply]
Hi Adel, I hope all is well with you.
We are still waiting for the translation's guidelines.

**Reply by boussadjra** [reply]
يجب أن نضع  قائمة للمصطلحات التقنية (glossary)   و أيضا للمصطلحات التي لا تحتاج  ترجمة مثل script setup او API ، حتى نوحد تلك المفردات ولا نقع في الغموض، هل تتفقون معي؟

**Reply by amans199** [reply]
I totally agree with you. How about we all meet next Sunday at (6:00 PM EET) to talk about the Arabic guidelines. 
@boussadjra @adel-habib

**Reply by boussadjra** [reply]
That's great @amans199

**Reply by Jamil-Bailony** [reply]
لدينا قاموسًا للمصطلحات التقنية وترجمنا العديد من التوثيقات التقنية ونُشرت في مكان واحد، موسوعة حسوب wiki.hsoub.com، يمكننا التعاون!

**Reply by amans199** [reply]
سيسعدنا التعاون معكم .. لو امكن تنضم معنا علي ديسكورد https://discord.gg/gKdgnQkU

**Reply by mmaksi** [reply]
I would love to participate in translating Vue.js docs into Arabic! Where are you meeting guys?

**Reply by faresdjefaflia** [reply]
> I would love to participate in translating Vue.js docs into Arabic! Where are you meeting guys?


https://discord.gg/gKdgnQkU

**Reply by abdurahman-harouat** [reply]
يبدو ان رابط ديسكورد لا يعمل لم استطع الدخول

**Comment by itsmolaboune** [comment]
hello guys, can i join to support our arabic language?

**Reply by boussadjra** [reply]
> hello guys, can i join to support our arabic language?

Yes of course

**Reply by itsmolaboune** [reply]
so let's make a group chat on discord and make tasks in trello to track what we do, did you agrees?

**Reply by amans199** [reply]
https://discord.gg/gKdgnQkU

**Comment by naif-sameer** [comment]
sound good. I want to join you 😎✌🏻. 
Let's make vue speak Arabic 🤩

**Reply by amans199** [reply]
https://discord.gg/gKdgnQkU

**Reply by boussadjra** [reply]
هذا رابط المناقشة الجديد https://github.com/vuejs-translations/guidelines/discussions/60

**Comment by amans199** [comment]
Whilst waiting for the guidelines.. how about we get to know each other and start planning.
Join our discord server below..
https://discord.gg/gKdgnQkU

**Reply by NidaaHamed** [reply]
invite invalid

**Comment by faresdjefaflia** [comment]
yes we can

**Comment by zeyadmoamen1210** [comment]
Hi guys i want to help 🤩
